### DISCUSSION TITLE
Mapping caps lock to escape and control depending on tap or chord

### DISCUSSION THREAD

**Discussion by mvcisback** [pop-os/cosmic-epoch#1152] [Q&A] [answered]
Is there a mechanism (current or planned) to have the caps lock key act as escape if it's used alone (for use in vim) and as control if held with another key?

**Comment by kurtlawrence** [comment, marked as answer]
I can't answer directly is the COSMIC team is looking into keyboard mapping support, but I would recommend checking out https://github.com/jtroo/kanata
I use it currently (with COSMIC) and it works really well.

**Reply by mvcisback** [reply]
Thanks. This is a nice work around.

It'd be nice to have such functionality built in, but if this works with cosmic, I'm happy to eplore this route.
